microshift_troubleshooting/microshift-troubleshoot-backup-restore.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="microshift-troubleshoot-backup-restore.adoc"]
+= Troubleshoot data backup and restore
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-troubleshoot-data-backup-and-restore
+
+toc::[]
+
+To troubleshoot failed data backups and restorations, check the basics first, such as data paths, storage configuration, and storage capacity.
+
+[id="troubleshoot-backup-restore-microshift-backup-data-failed"]
+== Backup data failed
+Data backups are automatic on OSTree systems. If you are not using an OSTree system and attempted to create a manual backup, the following reasons could cause the backup to fail:
+
+* If {product-title} stopped running because of an error, you cannot perform a backup of the data.
+** First, make sure the system is healthy.
+** Then, stop it in a healthy state before attempting a backup.
+** You might need to wait several minutes after a system start to successfully stop {product-title}. Allow the system to complete health checks and any other background processes before attempting to back up.
+
+* If you do not have sufficient storage for the data, the backup fails. Ensure that you have enough storage for the {product-title} data.
+
+* If you do not have sufficient permissions, a backup can fail. Ensure you have the correct user permissions to create a backup and perform the required configurations.
+
+[id="troubleshoot-backup-restore-microshift-backup-logs"]
+== Backup logs
+* With manual backups, the logs print to the console.
+* For RPM-OSTree automated backups, logs are automatically generated. You can check OSTRee logs by running the following command:
++
+[source, terminal]
+----
+$ sudo jounralctl -u microshift
+----
+
+[id="troubleshoot-backup-restore-microshift-restore-data-failed"]
+== Restore data failed
+The restoration of data can fail for a variety of reasons, including mismatched data versions and storage and permission issues. A few examples are provided here.
+
+[id="troubleshoot-backup-restore-microshift-RPM-OSTree-data-restore-failed"]
+=== RPM-OSTree systems data restore failed
+Data restorations are automatic on OSTree systems, but can fail, for example:
+* You are attempting to restore data from a deployment that no longer exists on your system.
+** Only the latest backups that have corresponding deployments are retained. Outdated backups that do not have a matching deployment are automatically removed.
+
+* You are attempting to restore data from a newer version of {product-title}.
```

Comment:
"You are" - maybe this section is not for "automated backup & restore" on "ostree-based systems" but for manual b&r on "regular RPM-based systems"?

```
=== RPM-OSTree systems data restore failed
```

If it's manual on non-ostree, then on non-ostree there's no notion of "Deployments"